docs/configuration/integrations/troubleshoot.md

```diff
@@ -0,0 +1,85 @@
+# Troubleshooting
+
+The sections below include information to help you troubleshoot issues that you may come across while configuring the integration of Codacy with your Git provider.
+
+If the information provided on this page is not enough to solve your issue, contact [support@codacy.com](mailto:support@codacy.com) providing all the information that you were able to collect while following the troubleshooting instructions.
+
+## GitHub Cloud and GitHub Enterprise authentication {id="github"}
+
+### 404 error
+
+While trying to authenticate on GitHub you get the following error message:
+
+![Invalid client id](images/github-invalid-client-id.png)
+
+This means that there is a mismatch in the Client ID that Codacy is using to authenticate on GitHub.
```

Comment:
Maybe here `This means` might be too strong. (now that I read it again :smile: )
We could probably say `This might mean`.